src/Akka.HealthCheck.Persistence.Tests/Akka.HealthCheck.Persistence.Tests.csproj

```diff
@@ -4,8 +4,12 @@
     <TargetFramework>netcoreapp2.1</TargetFramework>
     <IsPackable>false</IsPackable>
   </PropertyGroup>
+  <ItemGroup>
+    <Compile Include="..\Akka.HealthCheck.Tests\LiveNessProbeSpecBase.cs" Link="LiveNessProbeSpecBase.cs" />
+  </ItemGroup>
 
   <ItemGroup>
+    <PackageReference Include="akka.persistence.sqlite" Version="1.3.11" />
```

Comment:
Change the `Version` to read `$(AkkaVersion)` - that way we can synchronize on the Akka.NET version specific in `common.props`, just like how we do for the TestKit on the line below.